plugins/entity-feedback/src/components/FeedbackResponseTable/FeedbackResponseTable.tsx

```diff
@@ -80,9 +80,11 @@ export const FeedbackResponseTable = (props: FeedbackResponseTableProps) => {
       width: '35%',
       render: (response: ResponseRow) => (
         <>
-          {response.response?.split(',').map(res => (
-            <Chip key={res} size="small" label={res} />
-          ))}
+          {response?.response &&
+            response.response.length > 0 &&
+            response.response
```

Comment:
Previous suggestion to shorten this to: 
```
response.response?.length &&
            response.response
```
led to the following behavior
![Screenshot 2024-02-27 at 4 53 17 PM](https://github.com/backstage/backstage/assets/146387693/420d04db-63c3-4e1a-89a2-857ac667506b)

Reply:
hm ok how about `response.response?.length > 0 && response.response` ?

Reply:
What's up with that final `&& response.response`? After a length check. Is it an array or is it not? :)

Reply:
Could we ignore the length check in the first place? It's only gonna be split into an array immediately anyway, and rendering an empty array is fine and does nothing.

`(response.response || '').split(....`

Reply:
> Oh and as a general note. If you get a zero rendered, it's almost guaranteed to be a case of code on the form
> 
> ```tsx
> {condition && <Component />}
> ```
> 
> which is dangerous since _anything_ falsy will be sent for rendering (the `&&` evaluates to whatever was on the left side of the operator). And a zero is falsy but React happily renders it as a zero to the browser. This can generally be rewritten to
> 
> ```tsx
> {condition ? <Component /> : null}
> ```
> 
> to ensure that a null is what gets "rendered", not the outcome of the condition

ah TIL!

Reply:
That suggestion works as expected @freben, thank you! 
![Screenshot 2024-02-28 at 9 57 41 AM](https://github.com/backstage/backstage/assets/146387693/d486dde1-9bb5-4410-802d-baac52e39339)
![Screenshot 2024-02-28 at 9 57 30 AM](https://github.com/backstage/backstage/assets/146387693/a43a7e1c-9efc-4fe2-aed1-2ae31c0f909e)